Tools/CMake/JASP.cmake

```diff
@@ -92,19 +92,25 @@ message(CHECK_START "Checking for CRYPT_KEY")
 set(CRYPT_KEY
     ""
     CACHE STRING "")
+
 if(CRYPT_KEY STREQUAL "")
-  set(CRYPT_KEY $ENV{ENVIRONMENT_CRYPTKEY})
+	# lets see if the user set something in the environment
+  set(CRYPT_KEY $ENV{CRYPTKEY})
 
   if(CRYPT_KEY STREQUAL "")
-    message(CHECK_FAIL "not found.")
-    message(WARNING "CRYPT_KEY is not found.")
+	  message(CHECK_PASS "found default.")
+	  
+    message(WARNING "CRYPT_KEY is not found, using default replace if this is to go out to users!")
```

Comment:
I'd remove this warning. This is a very specific variable, and if someone wants to set it, they know what they are doing and this warning is only going to be distraction for most people.

Reply:
Fair enough